content/en/infrastructure/process/increase_process_retention.md

```diff
@@ -39,11 +39,11 @@ You can generate a new process-based metric directly from queries in the [**Live
 
 {{< img src="infrastructure/process/process2metrics_create.png" alt="Create a process-based metric" style="width:80%;">}}
 
-1. **Select tags to filter your query**: The query syntax is the same as for [Live Processes][2]. Only processes matching the scope of your filters are considered for aggregation. Text search filters are supported only on the Live Processes page.
+1. **Select tags to filter your query**: The available tags are the same as for [Live Processes][2]. Only processes matching the scope of your filters are considered for aggregation. Text search filters are supported only on the Live Processes page.
```

Comment:
The query syntax is not the same, boolean operators are not supported here, unlike in the Processes page.